ldk/javascript/src/whisper/index.ts

```diff
@@ -230,19 +238,30 @@ export type ChildComponents =
   | Telephone
   | TextInput;
 
-export declare type CollapseBox = WhisperComponent<WhisperComponentType.CollapseBox> & {
+export type CollapseBox = WhisperComponent<WhisperComponentType.CollapseBox> & {
   children: Array<ChildComponents>;
```

Comment:
If you add Box into the ChildComponents that will allow loop developer to add a Box inside of the CollapseBox. I thought requirement to allow Box in the Box was only for the Box component and not the CollapseBox.

Reply:
Ahhh - gotcha, OK, let me fix that

Reply:
Fixed!